ui/app/services/path-help.js

```diff
@@ -21,14 +24,25 @@ export default Service.extend({
     });
   },
 
-  getNewModel(modelType, backend, owner) {
+  //Makes a call to grab the OpenAPI document.
+  //Returns relevant information from OpenAPI
+  //as determined by the expandOpenApiProps util
+  getProps(helpUrl, path, backend) {
+    return this.ajax(helpUrl, backend).then(help => {
+      let props = help.openapi.paths[path].post.requestBody.content['application/json'].schema.properties;
```

Comment:
Do we know here that there will always be one path (at least until we get to more complex stuff with CRUD views, etc)? if so, what do you think about doing paths[0] instead of paths[path] ? When looking through the other code I wasn't sure what `path` was in those objects, but if this is all we're using it for, maybe we don't need it...

Reply:
paths is an object, so it's either this or doing something like
```
let path = Object.keys( help.openapi.paths)[0]
let props = paths[path].post.requestBody.content['application/json'].schema.properties;
```

Reply:
Ahh yeah I didn't realize that, but yeah what do you think about getting the keys if there's only going to be one?

Reply:
done and done